packages/rn-tester/RNTester/AppDelegate.mm

```diff
@@ -82,6 +82,12 @@ @implementation AppDelegate
 - (BOOL)application:(__unused UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
   RCTEnableTurboModule(YES);
+  // uncomment to configure NSURLSession
```

Comment:
We probably don’t need to add commented code here as this is an uncommon use case. Perhaps this explanation of usage should just live in the documentation.

Reply:
Alright, I can add a section about that. Would it be appropriate to add a section [here](https://reactnative.dev/docs/network) for instance?

Reply:
I think so

Reply:
I have created a PR that contains the needed documentation now: https://github.com/facebook/react-native-website/pull/2629

Reply:
Also, the commented code has been removed in: 1634c87

Reply:
@TheSavior Please see messages above 🙂